cns/restserver/util.go

```diff
@@ -40,6 +40,31 @@ func (service *HTTPRestService) setNetworkInfo(networkName string, networkInfo *
 	service.state.Networks[networkName] = networkInfo
 }
 
+func (service *HTTPRestService) SetPnpIDMacaddressMapping(ctx context.Context) error {
+	p := platform.NewExecClient(nil)
+	VfMacAddressMapping, err := platform.FetchMacAddressPnpIDMapping(ctx, p)
+	if err != nil {
+		return errors.Wrap(err, "failed to fetch MACAddressPnpIDMapping")
+	}
+	logger.Printf("[DEBUG] Macaddress mapping: %d", len(VfMacAddressMapping))
```

Comment:
Do we need this log line? This will be print out for both windows and linux

Reply:
No, I will remove this, just added now for debugging. we haven't tested it yet.